content/pt-br/docs/2.concepts/5.nuxt-lifecycle.md

```diff
@@ -1,108 +1,106 @@
-  - can only be defined in `store/index.js`
+  - Ação Vuex que é chamada apenas no lado do servidor para pré-carregar o store
+  - O primeiro argumento é o **Contexto Vuex**, o segundo argumento é o **Contexto Nuxt**.
+    - Nós despachamos outras ações daqui → é apenas o "ponto de entrada" para outras ações no lado do servidor
+  - só pode ser definido em `store/index.js`
 - Middleware
-  - Global middleware
-  - Layout middleware
-  - Route middleware
+  - Middleware Global
+  - Middleware do Layout
+  - Middleware de Rotas
 - asyncData
-- beforeCreate (Vue lifecycle method)
-- created (Vue lifecycle method)
-- The new fetch (top to bottom, siblings = parallel)
-- Serialization of state (`render:routeContext` Nuxt hook)
-
-- the HTML rendering happens (`render:route` Nuxt hook)
-
-- `render:routeDone` hook when HTML has been sent to the browser
-
-- `generate:before` Nuxt hook
-- HTML files are generated
-  - **Full static generation**
-    - e.g. static payloads are extracted
-  - `generate:page` (HTML editable)
-  - `generate:routeCreated` (Route generated)
-- `generate:done` when all HTML files have been generated
-
-### Client
-
-This part of the lifecycle is fully executed in the browser, no matter which Nuxt mode you've chosen.
-
-- Receives the HTML
-- Loading assets (e.g. JavaScript)
-- client-side Nuxt plugin
-  - in order as defined in nuxt.config.js
-- Vue Hydration
+- beforeCreate (Método do ciclo de vida do Vue)
+- created (Método do ciclo de vida do Vue)
+- Uma nova busca (de cima para baixo, em paralelo se forem vizinhos)
+- Serialização do estado (Hook`render:routeContext` do Nuxt)
+- A renderização do HTML acontece (Hook `render:route` do Nuxt)
+- Hook `render:routeDone`, quando o HTML foi enviado para o navegador
+- Hook `generate:before` do Nuxt
+- Arquivos HTML são gerados
+  - **Geração estática total**
+    - por exemplo, payloads estáticos são extraídos
+  - `generate:page` (HTML editável)
+  - `generate:routeCreated` (Rota gerada)
+- `generate:done` quando todos os arquivos HTML foram gerados
+
+### Cliente
+
+Esta parte do ciclo de vida é totalmente executada no navegador, não importa qual modo Nuxt você tenha escolhido.
+
+- Recebe o HTML
+- Carrega os assets (por exemplo, JavaScript)
+- Módulos Nuxt do lado do cliente
+  - em ordem conforme definido no `nuxt.config.js`
+- Hidratação do Vue
 - Middleware
-  - Global middleware
-  - Layout middleware
-  - Route middleware
-- asyncData (blocking)
-- beforeCreate (Vue lifecycle method)
-- created (Vue lifecycle method)
-- The new fetch (top to bottom, siblings = parallel) (non-blocking)
-- beforeMount (Vue lifecycle method)
-- mounted (Vue lifecycle method)
+  - Middleware Global
+  - Middleware do Layout
+  - Middleware de Rotas
+- asyncData (bloqueio)
+- beforeCreate (Método do ciclo de vida do Vue)
+- created (Método do ciclo de vida do Vue)
+- Uma nova busca (de cima para baixo, em paralelo se forem vizinhos) (não bloqueio)
```

Comment:
Uma nova busca -> O novo fetch

Reply:
Solved, but you can explain more about `The new Fetch`, what is this? What does this do? So I can understand if there is another translation that fits better.

Reply:
fetch is part of the lifecycle I believe, so no really something we want to translate